buildSrc/build.gradle

```diff
@@ -135,10 +135,10 @@ dependencies {
   api 'org.apache.maven:maven-model:3.6.2'
   api 'com.networknt:json-schema-validator:1.0.36'
   api "com.fasterxml.jackson.core:jackson-databind:${props.getProperty('jackson_databind')}"
-  api "org.bouncycastle:bcprov-jdk15to18:${props.getProperty('bouncycastle')}"
-  api "org.bouncycastle:bcpkix-jdk15to18:${props.getProperty('bouncycastle')}"
-  api "org.bouncycastle:bcpg-jdk15to18:${props.getProperty('bouncycastle')}"
-  api "org.bouncycastle:bcutil-jdk15to18:${props.getProperty('bouncycastle')}"
+  api "org.bouncycastle:bcprov-jdk18on:${props.getProperty('bouncycastle')}"
```

Comment:
```suggestion
  api "org.bouncycastle:bcprov-jdk15to18:${props.getProperty('bouncycastle')}"
```

Reply:
(and for others)

Reply:
Curious, how come? I cherry-picked https://github.com/opensearch-project/OpenSearch/pull/12317 which contained these changes in addition to the BC bump. We don't want to upgrade to the latest jdk18on for 1.3?

Reply:
I think we could, but for 1.x seems like safer to keep the artifacts unchanged (since all CVEs should be patched there as well):

> Java Version Details With the arrival of Java 15. jdk15 is not quite as unambiguous as it was. The jdk18on jars are compiled to work with anything from Java 1.8 up. They are also multi-release jars so do support some features that were introduced in Java 9, Java 11, and Java 15. If you have issues with multi-release jars see the jdk15to18 release jars below.

Reply:
Fair call, though all the license additions are for jdk18on 1.78?

Reply:
That is non issue I think, the licensees are the same (at least should be), the only diff is artifact name

Reply:
Seems it was an issue - I've updated them so should be good now.